### ISSUE TITLE
about nricp_amberg

### ISSUE THREAD

**Issue by datachiii** [mikedh/trimesh#2047]
The document shows that "steps" is the core of the algorithm, but the default is None, and in the paper "Amberg et al. 2007: Optimal Step Nonrigid ICP Algorithms for Surface Registration." the cost function contains ws, wl, wn, so these parameters should be necessary (except wl). If step is None, what are the values of ws, wl, wn? 
Are they all 0? 
If all are 0.  How does the cost function work?

**Comment by Kiord** [comment]
You are right, the algorithm cannot run without predefined "steps".  But if steps is None, a default hard coded value is set (see registration.py).

 This is not clear from the documentation and can be added. More detailed information is available in docs/guide/nricp.md. 

The hard coded default "steps" is meant make the algorithm work relatively well on most cases.

**Comment by datachiii** [comment]
Thank you for your reply. It helped me a lot.
